roles/config-minishift-remote/README.md

```diff
@@ -0,0 +1,60 @@
+Minishift Remote
+=========
+
+Ansible role to provision and setup a remote target for [Minishift](https://github.com/minishift/minishift).
+
+Requirements
+------------
+
+Access to an existing machine that can supports the use of `docker` and `firewalld` along with a user that can elevate themselves to root.
+
+Role Variables
+--------------
+
+This role contains a number of variables to customize the deployment of Clair. The following are some of the most important that may need to be configured
+
+| Name | Description | Default|
+|---|---|---|
+|docker_group|Group name for Docker|`docker`|
+|minishift_dependencies|Dependencies required to be present prior to configuring and starting Minishift| `docker, firewalld`| 
+|minishift_remote_user|Name of the user Minishift should connect as| Current user executing provisioning (`ansible_user`) |
+|minishift_user_ssh_public_key_file|Location of the public key file that should be allowed to connect to minishift| SSH key file used during ansible execution (`ansible_ssh_private_key_file`) or `~/.ssh/id_rsa.pub`|
+|minishift_firewalld_ports|Firewall ports to open | 80, 443, 2376, 4001 |
+|download_minishift|Download binary of Minishift | `true` |
+|minishift_install_dir|Location where Minishift binary should be downloaded | `/usr/local/bin` |
```

Comment:
@sabre1041 The only issue I had with this was having `minishift_install_dir` default to `/usr/local/bin` . Because I didn't provide a value -- it used that location and because I wasn't root -- it errored out on the untar. Once I changed it to a location that my user had access to -- everything was fine.

My thoughts are this -- either have that task executed as root so that it will have access to the default location regardless or have a better default that wouldn't require this (ex. I used /tmp and everything executed successfully.. but maybe there's a better location)